_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: Using Prometheus to visualize metrics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Using Prometheus to visualize metrics
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can visualize metrics data from the open-source [Prometheus](https://prometheus.io/) monitoring solution and from log data aggregated within OpenSearch, allowing you to analyze and correlate data across logs and traces as well as metrics. Previously, OpenSearch Observability allowed you to ingest and visualize logs and traces only from your monitored environments. With this feature, you can observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform your root cause analysis.
+
+## Creating a connection to Prometheus
+
+You can view metrics collected from Prometheus in OpenSearch Dashboards by first creating a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the SQL plugin. 
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
```

Comment:
If we have an image here, we need to describe it. You could pull this image above "Creating", just explain it. Like, 
The following image shows the process of pulling in metrics and viewing them in the Dashboard." (Something like that.)